bootstrap-cmake.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+# Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+usage() {
+    echo "Usage: $0 <source-dir> [<extra-cmake-args>]" >&2
+}
+
+if [[ $# -eq 1 ]]; then
+    SOURCE_DIR=$1
+    EXTRA_CMAKE_ARGS=""
+elif [ $# -eq 2 ]; then
+    SOURCE_DIR=$1
+    EXTRA_CMAKE_ARGS=$2
```

Comment:
Would expect that I could pass multiple extra cmake args like -DMYDEF1=something -DMYDEF2=somethingelse.

Reply:
Agree. Is there any easy way of varargs in bash?

Reply:
```
SOURCE_DIR=$1
shift
EXTRA_CMAKE_ARGS="$@"
```